tests/test_unit.py

```diff
@@ -257,13 +259,18 @@ def _build_budget_order_operations():
     expected_get_adwords_operation['fields'] = sorted(expected_get_adwords_operation['fields'])
     assert expected_get_adwords_operation == get_adwords_operation
 
+    start_date_time = datetime(2018, 12, 30, 23, 59, 59)
+    end_date_time = datetime(2037, 12, 30, 23, 59, 59)
+    time_zone = 'America/Sao_Paulo'
+
     mutate_internal_operation = {
         'object_type': 'budget_order',
         'client_id': 7857288943,
         'billing_account_id': '1234-5678-9012-3456',
         'primary_billing_id': '1234-5678-9012',
-        'start_date_time': '20361231 235959 America/Los_Angeles',
-        'end_date_time': '20371230 235959 America/Los_Angeles',
+        'start_date_time': json.loads(json.dumps(start_date_time, default=str)),
+        'end_date_time': json.loads(json.dumps(end_date_time, default=str)),
```

Comment:
As @sbneto pointed, is more explicit to call `process_json_datetime_to_adw_format` directly. Is strange to do `json.loads(json.dumps(...))`